src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo/UserInfo';
+import { Todo } from '../../types/todo';
+
+interface Props {
+  todo: Todo;
+}
+export const TodoInfo = ({ todo }: Props) => (
+  <article
+    data-id={todo.id}
+    className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}
+  >
+    <h2
+      className="TodoInfo__title"
+    >
+      {todo.title}
+    </h2>
+    <UserInfo
+      user={todo.user}
+    />
```

Comment:
```suggestion
    </h2>
    
    <UserInfo user={todo.user} />
```
Also, you can destructure all the needed props of todo

Reply:
fixed